ironfish/src/network/peers/peerManager.ts

```diff
@@ -322,9 +329,15 @@ export class PeerManager {
    * @param initiator Set to true if we are initiating a connection with `peer`
    */
   private initWebRtcConnection(peer: Peer, initiator: boolean): WebRtcConnection {
-    const connection = new WebRtcConnection(initiator, this.logger, this.metrics, {
-      simulateLatency: this.localPeer.simulateLatency,
-    })
+    const connection = new WebRtcConnection(
+      initiator,
+      this.logger,
+      this.stunServers,
```

Comment:
Can this go into the options below?

Reply:
Yea added it to options. Only reason I didn't before is because I didn't want to set a default value. After Andrea's comment I tested the node with and empty list of stunServers and it is still semi-functional so I think setting it to that is reasonable

Reply:
yes because it actually just defaults to built in servers, which is also fine